src/main/resources/lang_en_US.properties

```diff
@@ -137,4 +141,11 @@ CommandFactionMakePeaceRequested=Peace requested.
 FactionPeaceRequestSentNotificationTitle=Peace request sent
 FactionPeaceRequestSentNotificationBody={0} sent an offer of peace to {1}.
 FactionPeaceRequestReceivedNotificationTitle=Peace request received
-FactionPeaceRequestReceivedNotificationBody=You received an offer of peace from {0} of the faction {1}.
\ No newline at end of file
+FactionPeaceRequestReceivedNotificationBody=You received an offer of peace from {0} of the faction {1}.
+CommandFactionInfoNoPermission=You do not have permission to view faction information.
+CommandFactionInfoNotAPlayer=Only players can view faction information.
+CommandFactionInfoFailedToSavePlayer=Failed to save player while executing info command.
+CommandFactionInfoMustBeInAFaction=You must be in a faction to view faction information.
+CommandFactionInfoTitle==== {0} ===
```

Comment:
Does '=' work here or does it need to be escaped?
If you've tested this and it works, cool

Reply:
Yup this works!